phpunit-watcher.yml.dist

```diff
@@ -0,0 +1,9 @@
+watch:
+  directories:
+    - ./providers/
```

Comment:
Should we add the root directory to this list to ensure that changes to those files also trigger the runs?

Reply:
We could, as long as we exclude `vendor` etc. I'll look into that 👍🏻